### ISSUE TITLE
learning rate in optimization.py is so confusing

### ISSUE THREAD

**Issue by seongwook-ham** [google-research/ALBERT#49]
poly_power=1.0 is right for lamb or 0.5 is right?
in large batch optimization for deep learning training bert in 76 minutes (Y You et al) says poly power should be 0.5. but code says poly_power=1.0. which one is right?
and if i assume poly_power=1.0, init learning_rate 0.00176, learning_rate could not reach 0.00176 after warm up step, because during warm up statge, learning rate is already decayed to smaller value than 0.00176. am i right? 
what is real parameter used to train albert? i'm so confused

**Comment by ViktorAlm** [comment]
I might be in over my head here and you asked this question quite awhile back now so you might have figured it out. This is what I've found
![image](https://user-images.githubusercontent.com/1090762/72732118-5d515a80-3b95-11ea-80e4-2feedae88a44.png)

from
https://medium.com/@NvidiaAI/a-guide-to-optimizer-implementation-for-bert-at-scale-8338cc7f45fd

**Comment by seongwook-ham** [comment]
it helps a lot. thanks you
